channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt

```diff
@@ -32,15 +30,16 @@ open class JaicpReactions : Reactions() {
                 ""
             }
         }
-
-        return json {
+        val obj = buildJsonObject {
             if (this@JaicpReactions is TelephonyReactions) {
-                "dialer" to dialer.getApiResponse()
+                put("dialer", dialer.getApiResponse())
             }
-            "replies" to jsonArray {
-                jsonReplies.forEach { +it }
-            }
-            "answer" to answer
+            put("replies", buildJsonArray {
```

Comment:
There are ready to use `putJsonArray` and `putJsonObject` methods, that do exactly the same